src/components/PasswordField.jsx

```diff
@@ -1,19 +1,25 @@
 /* eslint-disable react/prop-types */
-import { useState } from "react";
+import React, { useState } from "react";
```

Comment:
I read that starting from React 17 it's no longer needed to import React for React components.

Reply:
I'm importing it because I'm referring directly to React in line 5

Reply:
In line 5 of `src/components/PasswordAlert.jsx` refers to React but not this file right?

Reply:
Right, I was looking at the wrong file - will change this